docker/worker/testdata/BLAH-127.yaml

```diff
@@ -10,9 +10,12 @@ affected:
 - package:
     name: blah.com/package
     ecosystem: golang
+  versions:
```

Comment:
There's a bit of a disconnect here in that we technically accept vulnerabilities with only git ranges and no versions arrays, because our infra will populate that if needed. 

We'll need to relax this rule here before we merge this: https://github.com/ossf/osv-schema/blob/92001ed827205e50509c8a286e298286674cc96d/validation/schema.json#L244

Reply:
Created a PR here: https://github.com/ossf/osv-schema/pull/93